src/Simplex/Chat.hs

```diff
@@ -2587,6 +2597,104 @@ processChatCommand' vr = \case
               let aciContent = ACIContent SMDRcv $ CIRcvGroupInvitation ciGroupInv {status = newStatus} memRole
               updateDirectChatItemView user ct itemId aciContent False Nothing
         _ -> pure () -- prohibited
+    sendContactContentMessage :: User -> ContactId -> Bool -> Maybe Int -> ComposedMessage -> Maybe CIForwardedFrom -> CM ChatResponse
```

Comment:
this, sendGroupContentMessage, createNoteFolderContentItem moved from api with a minimal change to support sending message as forwarded (see prepareMsg)